snippets/08_googlevideo.txt

```diff
@@ -14302,6 +14370,22 @@
 2a00:1450:4023:b::8 r3---sn-2gb7sn7r.googlevideo.com
 2a00:1450:4023:b::9 r4---sn-2gb7sn7r.googlevideo.com
 2a00:1450:4023:b::a r5---sn-2gb7sn7r.googlevideo.com
+2a0b:880:1::c r1---sn-a0jpm-a0ml.googlevideo.com

```

Comment:
GGC servers should be put together, before google servers. Line 4281 of `08_googlevideo.txt`